src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIDE_LENGTH = 100;
+    private static final int MAX_RADIUS = 50;
+    private static final int MAX_TRIANGLE_LEG = 100;
+    private static final int MAX_TRAPEZOID_BASE = 100;
+    private static final int MAX_TRAPEZOID_HEIGHT = 50;
+    private static final Random RANDOM = new Random();
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(10, Color.WHITE);
+    }
+
+    public Square getRandomSquare() {
+        double side = RANDOM.nextDouble(MAX_SIDE_LENGTH);
```

Comment:
The method 'nextDouble' in Random does not take any arguments. If you want to get a double within a specific range, you should use 'RANDOM.nextDouble() * MAX_SIDE_LENGTH'.